lib/fastly.rb

```diff
@@ -30,4 +33,10 @@ def self.purge_key(key, soft = false)
     Rails.logger.debug "Fastly purge url=#{url} status=#{json['status']} id=#{json['id']}"
     json
   end
+
+  def self.purge_api_cdn(gem_name)
+    ["deps/v1/#{gem_name}", "info/#{gem_name}", "versions", "names"].each do |path|
```

Comment:
At this point we should assume everything passes through Fastly. For `"deps/v1/#{gem_name}"`, it is not an actual endpoint, it's just a memcached key.